renovate.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "dependencyDashboard": true,
+  "extends": ["config:base"],
+  "timezone": "Europe/London",
+  "azure-pipelines": {
+    "enabled": true
+  },
+  "vulnerabilityAlerts": {
+    "groupName": "renovate",
+    "dependencyDashboardApproval": false,
+    "minimumReleaseAge": "0d",
+    "rangeStrategy": "update-lockfile",
+    "commitMessageSuffix": "[SECURITY]",
+    "branchTopic": "{{{datasource}}}-{{{depName}}}-vulnerability",
+    "prCreation": "immediate"
+  },
+  "packageRules": [
+    {
+      "matchDatasources": ["npm"],
+      "groupName": "Minor frontend dependencies",
+      "matchUpdateTypes": ["minor", "patch"],
+      "schedule": ["before 8am every weekday", "on the first day of the month"],
```

Comment:
How come using two schedule clauses here and elsewhere? I'd have thought "before 8am every weekday" is alone sufficient?

Reply:
As discussed I've reduced the granularity of the schedule and have changed the schedule to be `every month` / `every 3 months`